extensions/README.md

```diff
@@ -0,0 +1,8 @@
+# WebGPU Extensions
+
+Extensions defined in this directory are drafts and are **not part of the WebGPU specification**. Compliant implementations of WebGPU **must not** expose these extensions by default.
```

Comment:
```suggestion
Extensions defined in this directory are drafts and are **not part of the WebGPU specification**. Compliant implementations of WebGPU **must not** expose these extensions.
```

Reply:
(Putting an implementation in a mode that exposes these would be a non-compliant mode anyway)